examples/azure/terraform-azure-appserviceplan-example/main.tf

```diff
@@ -0,0 +1,56 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY AN AZURE APP SERVICE PLAN
+# This is an example of how to deploy an Azure App Service Plan
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
NIT: add reference to the corresponding test like we are doing here: https://github.com/gruntwork-io/terratest/blob/master/examples/azure/terraform-azure-nsg-example/main.tf#L1